TestCases/disc_adj_fsi/dyn_fsi/Primal/config.cfg

```diff
@@ -0,0 +1,28 @@
+SOLVER= MULTIPHYSICS
+MATH_PROBLEM= DIRECT
+CONFIG_LIST=(configFlow.cfg, configFEA.cfg)
```

Comment:
I think you can remove the files in Primal and FD_DV_0 right? (if they are not used for the testcase you are adding).

Reply:
Yes, these are now removed.